tests/deepspeed/test_zero.py

```diff
@@ -74,3 +74,15 @@ def test_stage2(self):
             }
         }
         self._check_zero(config)
+
+    @decorator.cuda_test
+    def test_stage2_with_grad_accumulation(self):
```

Comment:
You may need to copy the following content in json after rebase.
```
 'msamp': {
                'enabled': True,
                'opt_level': 'O3',
            },
            'zero_optimization': {
                'stage': 2,
            }
```

Reply:
Thanks! I have updated the config.